pkg/storage/tsdb/config.go

```diff
@@ -127,3 +130,12 @@ func (cfg *BucketStoreConfig) RegisterFlags(f *flag.FlagSet) {
 func (cfg *Config) BlocksDir(userID string) string {
 	return filepath.Join(cfg.Dir, userID)
 }
+
+// Validate the config
+func (cfg *BucketStoreConfig) Validate() error {
+	if cfg.SyncInterval <= 0 {
```

Comment:
I'd rather `syncInterval <= 0` be a disabled case. We currently rung with syncing disabled, and only send queries to ingesters in our testing, and it makes it more synonymous with Thanos where the LTS is optional.

Reply:
Got it. Will do.